src/main/interface/cli.c

```diff
@@ -936,12 +953,23 @@ static void cliSerialPassthrough(char *cmdline)
             if (strstr(tok, "tx") || strstr(tok, "TX"))
                 mode |= MODE_TX;
             break;
+#ifdef IOGetByTag
+        case 3:
+        	// When programming Arduino based devices such as MinimOSD, the DTR line is used to control
+        	// the reset. This parameter is the pin number. For example, 56 (0x38) is port C8. This allows
+        	// any GPIO to be used for driving DTR.
+            dtrPin = atoi(tok);
```

Comment:
Pin name parsing (`strToPin`) should be used (supplying `C8` as parameter)

Reply:
Excellent! Thanks for that.